lib/microbus/packager.rb

```diff
@@ -50,11 +51,25 @@ def run # rubocop:disable MethodLength
       file = fpm('.', fpm_opts)
 
       puts "Created #{file}"
+      checksum if opts.checksum
       file
     end
 
     private
 
+    # Generates a checksum of the build file and writes it to a new file.
+    #
+    # @return [String] MD5 checksum.
+    def checksum
```

Comment:
This method could probably use a more descriptive name.  It sounds as if it simply returns a value while the important bit is that it writes a file.